File: app/src/main/java/br/com/patrocine/patrocine/ui/activities/SplashScreenActivity.java
package br.com.patrocine.patrocine.ui.activities;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.widget.TextView;

import br.com.patrocine.patrocine.BuildConfig;
import br.com.patrocine.patrocine.R;

public class SplashScreenActivity extends Activity implements Runnable {

    private static final String TAG = SplashScreenActivity.class.getSimpleName();
    private static final long delay = 1500L;
    TextView tvVersion;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);
        new Handler().postDelayed(this, delay);

        onCreateComponents();
    }

    @Override
    protected void onPostCreate(Bundle savedInstanceState) {
        super.onPostCreate(savedInstanceState);
    }

    @Override
    public void run() {
        //startActivity(new Intent(this, MainActivity.class));
        startActivity(new Intent(this, MoviesTest.class));
        finish();
    }

    void onCreateComponents(){
        tvVersion = findViewById(R.id.version);
        String text = "version: " + BuildConfig.VERSION_NAME;
        tvVersion.setText(text);
    }
}

File: app/src/main/java/br/com/patrocine/patrocine/ui/activities/SplashScreenActivity.kt
package br.com.patrocine.patrocine.ui.activities

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.os.Handler
import br.com.patrocine.patrocine.BuildConfig
import br.com.patrocine.patrocine.R
import kotlinx.android.synthetic.main.activity_splash_screen.*

class SplashScreenActivity : Activity(), Runnable {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)
        Handler().postDelayed(this, delay)

        onCreateComponents()
    }

    override fun onPostCreate(savedInstanceState: Bundle?) {
        super.onPostCreate(savedInstanceState)
    }

    override fun run() {
        startActivity(Intent(this, WelcomeActivity::class.java))
        // startActivity(new Intent(this, MoviesTest.class));
        finish()
    }

    internal fun onCreateComponents() {
        val text = "version: " + BuildConfig.VERSION_NAME
        version.text = text
    }

    companion object {

        private val TAG = SplashScreenActivity::class.java.simpleName
        private val delay = 1500L
    }
}